### ISSUE TITLE
zwave-js-ui does not start on Rasbian 12 - cannot load libatomic.so.1

### ISSUE THREAD

**Issue by rglory** [giaever-online-iot/zwave-js-ui#7]
node: error while loading shared libraries: libatomic.so.1: cannot open shared object file: No such file or directory

Package is installed:
libatomic1 is already the newest version (12.2.0-14+rpi1).

And googling this problem shows that is a problem with a snap.

Any help?

PS Switching to stable channel fixed the issue. Looks like this is problem of "candidate" version.

**Comment by robertsLando** [comment]
@jmgiaever it should be as easy as adding this dep to the snapcraft: `apt-get install libatomic1`

**Comment by jmgiaever** [comment]
> apt-get install libatomic1

This won't help. But staging it as I've done it for the amrhf should do it. 

I can't remember any longer why it was necessary for armhf in the past, but not arm64 (and amd64). 

However it is strange if this is happening just on Raspbian? 

It runs flawlessly on both Ubuntu Server arm64 and Ubuntu Core arm64. 

```
sudo snap install zwave-js-ui --candidate
```

@rglory You've followed the setup instructions for rasbian?

**Comment by rglory** [comment]
@jmgiaever > You've followed the setup instructions for rasbian?

 Yes I did and this is not a new install. It used to work, but quite recently (though difficult to say exactly say when) stopped. I started to investigate and found this issue. Switching back to stable (I had to use candidate when upgraded home assistant) fixes this issue. And if I look through virtual filesystems I do see libatomic.so.1 now:

/snap/zwave-js-ui/287/usr/lib/arm-linux-gnueabihf/libatomic.so.1

When it was on candidate it was missing.
